src/test/java/sg/dex/starfish/integration/keeper/SquidResolverTest.java

```diff
@@ -0,0 +1,37 @@
+package sg.dex.starfish.integration.keeper;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import sg.dex.starfish.impl.squid.SquidResolverImpl;
+import sg.dex.starfish.util.DID;
+
+import org.web3j.crypto.CipherException;
+import java.io.IOException;
+
+public class SquidResolverTest {
+    private  SquidResolverImpl resolver;
+    private String value1;
+    private String value2;
+    private DID did;
+
+    public SquidResolverTest() throws IOException, CipherException {
+        resolver = SquidResolverImpl.create();
```

Comment:
Minor style point: if setting in constructor can be easier to just define these in the field declarations above.